src/test/java/de/qabel/desktop/hockeyapp/VersionClientTest.java

```diff
@@ -0,0 +1,200 @@
+package de.qabel.desktop.hockeyapp;
+
+import de.qabel.core.accounting.CloseableHttpClientStub;
+import de.qabel.core.accounting.CloseableHttpResponseStub;
+import org.apache.http.NameValuePair;
+import org.apache.http.entity.BasicHttpEntity;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONException;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class VersionClientTest {
+
+    private static final int VERSION_ID_1_1 = 208;
+    private static final int VERSION_ID_1_0 = 195;
+
+    CloseableHttpClientStub httpClient = new CloseableHttpClientStub();
+    private HockeyAppConfiguration config = new HockeyAppConfiguration("1.1", httpClient);
+    private VersionClient client = new VersionClient(config, httpClient);
+
+    @Test
+    public void checkAppVersion() {
+        assertEquals("1.1", config.getAppVersion());
```

Comment:
There is no need to test getters. If this test is the only code that uses `getAppVersion()` then just remove that getter. If not, test the code that uses the getter.